cfy_manager/components/mgmtworker/scripts/allow-snapshot-ssl-client-cert-access

```diff
@@ -0,0 +1,3 @@
+#! /usr/bin/env bash
+chown cfyuser. /etc/cloudify/ssl/stage_db.*
+chown cfyuser. /etc/cloudify/ssl/composer_db.*
```

Comment:
this is just two items, yes? the cert and the key?
i'm somewhat afraid of globs in sudo, maybe let's just write out the two files?

Reply:
This isn't in sudo, it's in the script hat is being run already through sudo- but yes, point taken.